.github/CODEOWNERS

```diff
@@ -1,5 +1,5 @@
 # Require review from domain experts when the PR modified significant config files.
 /server/config/config.go @tikv/pd-configuration-reviewer
-/server/schedule/schedulers/hot_region_config.go @tikv/pd-configuration-reviewer
+/pkg/schedule/schedulers/hot_region_config.go @tikv/pd-configuration-reviewer
```

Comment:
“This CODEOWNERS file contains errors”

Reply:
This PR is for migration only, no other changes.